MAVProxy/modules/mavproxy_console.py

```diff
@@ -653,6 +655,13 @@ def mavlink_packet(self, msg):
                     print(ex)
                     pass
 
+    def idle_task(self):
+        now = time.time()
+        if self.last_unload_check_time + self.unload_check_interval < now:
```

Comment:
It's usually better to subtract times as a habit rather than adding them.  Doesn't really matter here, but sometimes it does.